src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -196,6 +197,26 @@ public final void onCompleted(final Run run, @Nonnull final TaskListener listene
           statsd = getDescriptor().leaseClient();
           statsd.incrementCounter("completed", tagsToCounter);
           logger.fine(String.format("Attempted to send 'completed' counter with tags: %s", Arrays.toString(tagsToCounter)));
+
+          // Send KPIs 
+          if (!isMasterBranch(run)) {
+            if (run.getResult() == Result.SUCCESS) {
+              long mttr = getMeanTimeToRecovery(run);
+              long cycleTime = getCycleTime(run);;
+              long leadTime = run.getDuration() + mttr;
+
+              statsd.gauge("cycletime", cycleTime / 1000, tagsToCounter);
```

Comment:
Small thing: can you rename `tagsToCounter` to something like `statsdTags`?

Reply:
No problem